protocols/modbus/src/main/resources/protocols/modbus/modbus.mspec

```diff
@@ -249,7 +249,7 @@
 [type 'ModbusPDUReadFileRecordResponseItem'
     [implicit   uint 8     'dataLength'     '(COUNT(data) * 2) + 1']
```

Comment:
I think this needs to be adjusted to "COUNT(data) + 1" as the datatype changed from short to byte